### DISCUSSION TITLE
Version after correction for Share is interrupting web use

### DISCUSSION THREAD

**Discussion by machiyag** [Navid200/xDrip#153] [Dexcom]
I installed the 12/8 (I think, but see no way to confirm that) nightly build, and can again get readings. Every time a reading comes in, it interrupts whatever web page I am on. The last one was while I was writing testimony on a government proposal, and the interruption erased my entire testimony. This is a big problem.

**Comment by Navid200** [comment]
There is already a thread about this: https://github.com/Navid200/xDrip/discussions/150

**Reply by machiyag** [reply]
You've linked them. Should I (try to) move this post?

**Reply by Navid200** [reply]
You don't need to.

**Comment by machiyag** [comment]
Thanks.

mh

**Comment by Navid200** [comment]
What phone is xDrip on?

**Comment by Navid200** [comment]
The title of this thread may be taken as only the version after the Dexcom share fix is causing this problem.
Is that really the case?

**Comment by machiyag** [comment]
Pixel 3a, running Android 12.

mh

**Comment by machiyag** [comment]
I believe I tried the 12/2 version unsuccessfully, and loaded the 12/8,
which worked. That's what I'm currently running.

mh

**Reply by Navid200** [reply]
But, my question is about the apps taking you out of the page you were on when xDrip performs a reading.

Does that happen only with one version of xDrip?
If yes, we need to know for sure that you don't experience the issue with the previous versions.
If you do experience this with all versions of xDrip, the title of this thread could be misleading.

**Comment by machiyag** [comment]
It never happened until Dexcom messed with Share, and xDrip accommodated
the change.

I can't say it *wasn't* in the 12/2 nightly build, but it certainly is
happening with the 12/8. I'm sorry I don't have more detail.

mh
